src/components/ArticleList.vue

```diff
@@ -85,6 +85,15 @@ export default {
         ...Array(Math.ceil(this.articlesCount / this.itemsPerPage)).keys()
       ].map(e => e + 1);
     },
+    coolFilteredArticles() {
+      const articles = [];
+      this.articles.forEach(article, function() {
+        if (article.isCool) {
+          articles.push(article);
+        }
+      });
+      return articles;
```

Comment:
You can shorten this by using `Array.filter`
```suggestion
      return this.articles.filter((article) => article.isCool)
```